app/components/avo/fields/edit_component.rb

```diff
@@ -12,8 +12,9 @@ class Avo::Fields::EditComponent < ViewComponent::Base
   attr_reader :resource
   attr_reader :stacked
   attr_reader :view
+  attr_reader :autofocus
```

Comment:
`attr_reader` is slower than directly using the `@autofocus` instance variable and since we don't want to access it outside the component we should use directly the instance variable.

```suggestion
```

Reply:
Yes, my mistake.